src/helpers/pressable-functions.php

```diff
@@ -76,6 +75,9 @@ function get_related_pressable_sites( object $site, ?callable $node_generator =
 
 	// Identify the related sites by level.
 	$all_sites = get_pressable_sites();
+	if ( ! \is_array( $all_sites ) ) {
+		return null;
```

Comment:
Just for a consistent return type, might this be better to return as an empty array?

Reply:
If we were using PHP8+, I'd probably make the return type `array|WP_Error` but since we're on 7.4, I used the `null` as an indication that there's been an error. Having a nullable return type is consistent with all the other helpers, and this function's return type was already `?array`.

We can definitely infer that an empty array means that an error happened, but that's only because we know that our account wouldn't be empty. If someone would change the Pressable API credentials to point to an empty account (for testing something, for example), that logic would break. So I believe that returning `null` for error is the best solution in PHP7.4. 

Thoughts?